### PR TITLE
Add custom module to migrate Drupal 8 terms to Drupal 8

### PR DESCRIPTION
**Pull request by langelhc**
Instructions to test in README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
 # drupal8-migration-terms
-Module to Migrate Drupal 8 terms to Drupal 8
+
+Module to migrate Drupal8 terms to Drupal 8
```
Comment (enzolutions) [inline README.md]: Please add and space between Drupal and 8

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+
+Instructions:
+
+- Add credentials to settings.php file:
```
Comment (enzolutions) [inline README.md]: Please include instruction than explaining its information from Drupal 8 source database

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+  'password' => 'root',
+  'host' => 'mariadb',
+  'port' => '3306',
+  'prefix' => '',
+  'collation' => 'utf8mb4_general_ci',
+);
+
+- Enable 'drupal8_migration_terms' custom module and its dependencies.
+
+$ drush en drupal8_migration_terms -y
```
Comment (enzolutions) [inline README.md]: Use Drupal Console instead of Drush and link to drupalconsole project

File: README.md [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+
+- Execute migration:
+
+$ drush mi migration_term_tags
```
Comment (enzolutions) [inline README.md]: Use Drupal Console

File: README.md (hunk 2 above)
Comment (langelhc) [reply README.md]: I added an example, https://github.com/weknowinc/drupal8_migration_terms/pull/1/commits/f5c9bc13045cd6283b0754a471184284f1f972ec

File: README.md (hunk 1 above)
Comment (langelhc) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (langelhc) [reply README.md]: Done

**Pull request merged**